index.html

```diff
@@ -0,0 +1,269 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <title>Asad Shareef - Portfolio</title>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <link href="style.css" rel="stylesheet">
+</head>
+
+<body>
+
+  <header>
+    <nav class='navigation'>
+      <a href="index.html" class="title-top">Welcome</a>
+      <div class="nav-button">
+        <img src="assets/Union.png" />
+      </div>
+    </nav>
+  </header>
+
+
+  <section class="content">
+    <section class="title-body">
+      <h1>Hey there. I'm Asad</h1>
+      <h2 class='highlited-title'>I'm a software developer</h2>
+    </section>
+
+    <p class='about-me-brief'>
+      I can help you build a product , feature or website Look through some of my work and experience! If you like
+      what
+      you see and have a project you need coded,
+      don’t hestiate to contact me.
+    </p>
+
+    <section class='social-icons'>
+      <ul>
+        <li>
+          <a target="_blank" href="#"><img src="assets/twitter.png"
+              alt="Twitter" /></a>
+        </li>
+        <li>
+          <a target="_blank" href="#"><img src="assets/Linkedin icon.png"
+              alt="LinkedIn" /></a>
+        </li>
+        <li>
+          <a target="_blank" href="#"><img src="assets/github.png" alt="GitHub" /></a>
+        </li>
+        <li>
+          <a target="_blank" href="#"><img src="assets/medium.png"
+              alt="GitHub" /></a>
+        </li>
+        <li>
+          <a target="_blank" href="#"><img src="assets/angellist.png"
+              alt="GitHub" /></a>
+        </li>
+      </ul>
+    </section>
+
+    <section class='button-down'>
+      <a target="_blank" href="#"><img src="assets/disabled.png" alt="down" /></a>
+    </section>
+  </section>
+
+  <section class="projects">
+    <section class='recent-works'>
+      <h1 class="my-recent-works">My Recent Works</h1>
+      <hr class="hr-works">
+    </section>
```

Comment:
- [ ] Remove these `section` tags around here since they don't have any semantic meaning.

Reply:
@christianbiring1 
I updated the section tag from +64 to +69.